cfme/tests/test_replication.py

```diff
@@ -291,4 +292,33 @@ def test_replication_subscription_revalidation_pglogical():
             1. Validation succeeds as this subscription was successfully
                saved and is currently replicating
     """
-    pass
+
+    remote_app, global_app = configured_appliance, unconfigured_appliance
+    app_params = dict(
+        region=99,
+        dbhostname='localhost',
+        username=credentials["database"]["username"],
+        password=credentials["database"]["password"],
+        dbname='vmdb_production',
+        dbdisk=global_app.unpartitioned_disks[0],
+        fetch_key=remote_app.hostname,
+        sshlogin=credentials["ssh"]["username"],
+        sshpass=credentials["ssh"]["password"],
+    )
+
+    global_app.appliance_console_cli.configure_appliance_internal_fetch_key(**app_params)
+    global_app.evmserverd.wait_for_running()
+    global_app.wait_for_web_ui()
+
+    remote_app.set_pglogical_replication(replication_type=':remote')
+    region = global_app.collections.regions.instantiate(number=99)
+    region.replication.set_replication(replication_type="global",
+                                       updates={"host": remote_app.hostname},
+                                       validate=True)
+
+    view = region.replication.create_view(ReplicationGlobalView)
```

Comment:
Check out the PRT results, the test is failing at this point, and the screenshot shows that the replication role was not saved.